src/ScottPlot/Palette.cs

```diff
@@ -24,6 +24,7 @@ public static class Palette
         public static ScottPlot.Drawing.Palette OneHalfDark => new(new ScottPlot.Drawing.Colorsets.OneHalfDark());
         public static ScottPlot.Drawing.Palette PolarNight => new(new ScottPlot.Drawing.Colorsets.PolarNight());
         public static ScottPlot.Drawing.Palette SnowStorm => new(new ScottPlot.Drawing.Colorsets.Snowstorm());
+        public static ScottPlot.Drawing.Palette xgfs25 => new(new ScottPlot.Drawing.Colorsets.xgfs25());
```

Comment:
I think if we don't have a better name then either "Anton" or "Tsitsulin" (after the inspiration) would be a good idea.